src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.mapper.impl.response.AuthenticationResponseMapper;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+    private final AuthenticationResponseMapper authenticationResponseMapper;
+
+    @Autowired
+    public AuthenticationController(AuthenticationService authenticationService,
+                                    AuthenticationResponseMapper authenticationResponseMapper) {
```

Comment:
```suggestion
                                    DtoResponseMapper<UserResponseDto, User> authenticationResponseMapper) {
```
change in each controller